Source/WebCore/platform/graphics/cocoa/FontCacheCoreText.cpp

```diff
@@ -778,6 +779,7 @@ RetainPtr<CTFontRef> preparePlatformFont(CTFontRef originalFont, const FontDescr
 
     addAttributesForInstalledFonts(attributes.get(), fontDescription.shouldAllowUserInstalledFonts());
 
+    // todo: font-size-adjust: I can maybe add adjusted size here instead
```

Comment:
This project uses `FIXME`, not `todo`. Also, it would be good to have an easier to understand comment here. Like "can maybe add" is pretty ambiguous. Could you write down why you think it might be needed to work correctly?